app/public/receiver.js

```diff
@@ -12,8 +12,18 @@ const getCurrentRoom = () => {
 
 WS.onConnect(() => {
     console.log("We are connected");
-    getCurrentRoom();
 });
+
+const registerClient = () => {
+    if (WS.webSocket.readyState) {
+        getCurrentRoom();
```

Comment:
We'll eventually want to replace this with a `getCurrentState()` call, since there may be many properties of an existing game that a re-connecting player needs to know